interface/main/finder/multi_patients_finder.php

```diff
@@ -0,0 +1,258 @@
+<?php
+// Copyright (C) 2017 Amiel Elboim <amiele@matrix.co.il>
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
```

Comment:
Needs the standard license/author/copyright header

Reply:
Done